### PR TITLE
Fix stress

### PR DESCRIPTION
**Pull request by anne-glerum**
As first mentioned on the forum by Sungho, the stress should be computed as `2*eta*strain_rate-P*I` or `-2*eta*strain_rate+P*I`, instead of `2*eta*strain_rate+P*I` as is currently done in the visualization postprocessors that output stresses. Here I fixed the sign, sticking to the convention of positive compressive stresses, so `-2*eta*strain_rate+P*I`.

See also discussion in #3878 .

### For all pull requests:

* [x] I have followed the [instructions for indenting my code](../blob/master/CONTRIBUTING.md#making-aspect-better).

### For new features/models or changes of existing features:

* [ ] I have tested my new feature locally to ensure it is correct.
* [ ] I have [created a testcase](http://www.math.clemson.edu/~heister/manual.pdf#sec%3Awriting_tests) for the new feature/benchmark in the [tests/](../blob/master/tests/) directory.
* [x] I have added a changelog entry in the [doc/modules/changes](../blob/master/doc/modules/changes) directory that will inform other users of my change.

### CODE REVIEW

**Review by naliboff — COMMENTED** [review]
Looks good, thanks @anne-glerum! The only key remaining item would be to update the test output.

A general questions: Is there somewhere in the manual that discusses visualizing stress that should be updated?

Once merged, the forum post should also be updated.

File: source/postprocess/visualization/shear_stress.cc
Comment (bobmyhill) [inline source/postprocess/visualization/shear_stress.cc:77]: The "compressible_strain_rate" variable is actually the incompressible part of the strain rate. Ditto for the other two files. Suggest changing the name to incompressible_strain_rate?

This is my only quibble :)

Comment (anne-glerum) [reply source/postprocess/visualization/shear_stress.cc:77]: Thanks for having a look. It depends on what the material model returns for `(this->get_material_model().is_compressible()` whether the strain rate in `compressible_strain_rate` is in fact compressible or the incompressible part. So the name is not always correct. But that's not really related to this PR, so I'll leave that for now, OK?

**Comment by anne-glerum** [comment]
* Alright, I've checked that the failing tests failed because of the stress postprocessors and updated the results.
* I couldn't find an instance in the manual about stress visualization that needs to be updated. 
* I'll update the forum post when this is merged.

File: source/postprocess/visualization/shear_stress.cc
Comment (naliboff) [reply source/postprocess/visualization/shear_stress.cc:77]: Hmm, the relevant code above is:

>   = (this->get_material_model().is_compressible()
>                  ?
>                  strain_rate - 1./3 * trace(strain_rate) * unit_symmetric_tensor<dim>()
>                  :
>                  strain_rate);

Even if the material is compressible, the pressure is removed (-1/3 * trace(strain_rate)) and thus you have the deviatoric strain rate. If an incompressible formulation is used, then should the strain rate by default be deviatoric?

If the above is correct, would `deviatoric_strain_rate` be the most accurate name?

No preference on whether this is resolved now or in a follow-up PR, but will leave that up to @bobmyhill.

Comment (bobmyhill) [reply source/postprocess/visualization/shear_stress.cc:77]: @naliboff Yes, that's what I was trying to say. Thanks for clarifying!
I'm fine for the rename to be a separate PR.

**Review by naliboff — APPROVED** [review]
@anne-glerum - This is good to go, thanks! I'll make a follow-up PR that renames `compressible_strain_rate` to 'deviatoric_strain_rate', per @bobmyhill suggestion.

**Pull request merged**
